webui/react/src/themes.ts

```diff
@@ -154,26 +158,38 @@ export const lightTheme: Theme = {
 };
 
 const stateColorMapping = {
-  [RunState.Active]: 'var(--theme-colors-states-active)',
-  [RunState.Canceled]: 'var(--theme-colors-states-inactive)',
-  [RunState.Completed]: 'var(--theme-colors-states-success)',
-  [RunState.Deleted]: 'var(--theme-colors-states-failed)',
-  [RunState.Errored]: 'var(--theme-colors-states-failed)',
-  [RunState.Paused]: 'var(--theme-colors-states-suspended)',
-  [RunState.StoppingCanceled]: 'var(--theme-colors-states-inactive)',
-  [RunState.StoppingCompleted]: 'var(--theme-colors-states-success)',
-  [RunState.StoppingError]: 'var(--theme-colors-states-failed)',
-  [CommandState.Pending]: 'var(--theme-colors-states-suspended)',
-  [CommandState.Assigned]: 'var(--theme-colors-states-suspended)',
-  [CommandState.Pulling]: 'var(--theme-colors-states-active)',
-  [CommandState.Starting]: 'var(--theme-colors-states-active)',
-  [CommandState.Running]: 'var(--theme-colors-states-active)',
-  [CommandState.Terminating]: 'var(--theme-colors-states-inactive)',
-  [CommandState.Terminated]: 'var(--theme-colors-states-inactive)',
+  [RunState.Active]: 'active',
+  [RunState.Canceled]: 'inactive',
+  [RunState.Completed]: 'success',
+  [RunState.Deleted]: 'failed',
+  [RunState.Errored]: 'failed',
+  [RunState.Paused]: 'suspended',
+  [RunState.StoppingCanceled]: 'inactive',
+  [RunState.StoppingCompleted]: 'success',
+  [RunState.StoppingError]: 'failed',
+  [CommandState.Pending]: 'suspended',
+  [CommandState.Assigned]: 'suspended',
+  [CommandState.Pulling]: 'transitioning',
+  [CommandState.Starting]: 'transitioning',
+  [CommandState.Running]: 'active',
+  [CommandState.Terminating]: 'inactive',
```

Comment:
Still in the early phases of discussing updating states, but we are looking at adding a queued stated and grouping `Pulling` and `Starting` into a `Preparing` state. Which will come before `Active`. As far as color goes, we should keep this basic so we don't get into a crazy rainbow color scheme to cover all the edge cases. I think it would be best to keep the color the same as Active when it's in the state of "performing something".

Reply:
https://github.com/determined-ai/determined/pull/1094/commits/6f6b65d199ca46c9c0de4c524d97bc4fe33b4a13